extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/IcebergCommands.scala

```diff
@@ -94,7 +95,12 @@ object IcebergCommands extends Enumeration {
         inputObjs: ArrayBuffer[PrivilegeObject],
         outputObjs: ArrayBuffer[PrivilegeObject]): Unit = {
       this.buildInput(plan, inputObjs, commandTypes)
-      this.buildOutput(plan, outputObjs, commandTypes, outputActionType)
+      this.buildOutput(
+        plan,
+        outputObjs,
+        operationType != CREATETABLE,
```

Comment:
I am not sure whether it will become a upstream dependency with specific exclusion for all commands privilege builder. I prefer default value and let each command decides.

Reply:
Updated as suggested.